modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/search/spi/model/index/contributor/CommerceOrderModelDocumentContributor.java

```diff
@@ -48,6 +50,10 @@ public void contribute(Document document, CommerceOrder commerceOrder) {
 				_commerceChannelLocalService.getCommerceChannelByOrderGroupId(
 					commerceOrder.getGroupId());
 
+			CommerceOrderType commerceOrderType =
```

Comment:
orderType is not mandatory for a CommerceOrder so this could fail.
You must use a fetch and just after do a null check

Reply:
Thanks Luca